examples/systemcalls/systemcalls.c

```diff
@@ -93,6 +147,55 @@ bool do_exec_redirect(const char *outputfile, int count, ...)
  *
 */
 
+    pid_t fork_status;
+    int exec_status = 0;
+    int wait_status = 0;
+    int status;
+
+    int fd_status = open(outputfile, O_WRONLY | O_CREAT | O_TRUNC, 0644);
```

Comment:
@AshwiniCUB think about how you could avoid duplicating logic between these two functions.